query/language/define.feature

```diff
@@ -937,17 +1032,23 @@ Feature: TypeQL Define Query
       | datetime   | graduation-date   |
 
 
-  # TODO
-#  Scenario Outline: a type can own a '<value_type>' attribute type as a key
```

Comment:
I just removed these `type_manager`-specific cases, because it is covered in concept BDDs. I didn't clean the existing working tests as I consider it a sufficient amount of duplicated tests to ensure that we follow the same logic through typeql execution (at least until we have the generated typeql queries based on the concept api steps we discussed earlier).